tools/meteor-services/deploy.js

```diff
@@ -144,7 +144,8 @@ var authedRpc = function (options) {
   var infoResult = deployRpc({
     operation: 'info',
     site: rpcOptions.site,
-    expectPayload: []
+    expectPayload: [],
+    qs: options.qs
```

Comment:
I would think if anything that's desirable so that Galaxy could always induce a message in the CLI without requiring sending the entire bundle to the server.